gf-guide/metadata.md

```diff
@@ -389,3 +369,62 @@ Example:
 ``` code
 minisite_url: "https://myfontwebsite.com"
 ```
+
+
+### Classification
+
+TO DO
+
+### Stroke
+
+TO DO
+
+### Sample text
+
+This key overrides the sample texts provided in the languages texprotos (that you can find in the [Lang](https://github.com/googlefonts/lang/tree/main/Lib/gflanguages/data/languages) repository).
+
+If sample text is given within a sample text group (e.g. poster, specimen), all fields within that group must be provided.
+
+- `masthead_full` is a 4 glyphs string (e.g. AaBb) used in a single-script masthead. It is used in specimen masthead component.
+- `masthead_partial` is a 2 glyphs string (e.g. AaBb) that contributes in building a multi-script masthead. It is used in specimen masthead component.
```

Comment:
The example here still shows 4 glyphs. Should they be literally only 2?

Reply:
I copy-pasted Nate's document and it seemed odd to me too

Reply:
I asked back Nathan on the chat but didn't get any answer… So probably ok to leave it like that for now